build.gradle

```diff
@@ -217,6 +217,7 @@ sonarqube {
     property "sonar.exclusions", sonarExclusions.join(', ')
     property "sonar.coverage.exclusions", sonarExclusions.join(', ')
     property "sonar.coverage.jacoco.xmlReportPaths", "${jacocoTestReport.reports.xml.destination.path}"
+    property "sonar.cpd.exclusions", "**/uk/gov/hmcts/reform/hmc/api/controllers/*.java"
```

Comment:
any specific reason for adding this exclusion?

Reply:
Had some sonar qube error, in order to fix it , I added.